packages/components/src/utils/index.tsx

```diff
@@ -4,9 +4,18 @@ import {
   PressableStateCallbackType,
   useColorScheme as RNUseColorScheme,
   StyleProp,
+  View,
   ViewStyle,
 } from 'react-native'
-import { useEffect, useState } from 'react'
+import { Colors as TokenColors } from '@department-of-veterans-affairs/mobile-tokens'
+import React, { FC, useEffect, useState } from 'react'
+
+/** Function to prefill base gray colors */
+export function Base() {
```

Comment:
This utility function probably won't be needed once we get the semantic tokens/theming in, but for now can we use a more descriptive name for this? Maybe something like `BaseColor`.

Reply:
Hmmm, I created the function because I wasn't sure we'd be exposing that--I know that would sort things for our components, but would it be implemented in a way that was easy for an app to pull in to apply the same colors to their custom content sent in within the `children`?

If so, maybe should have a TODO to remove this when that comes.

I can rename it to be a bit more descriptive in the meantime.